unity-renderer/Assets/DCLPlugins/WorldsPlugin/WorldsPlugin.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using DCL;
+using Decentraland.Bff;
+using UnityEngine;
+using Variables.RealmsInfo;
+
+public class WorldsPlugin : IPlugin
+{
+
+    internal List<IWorldsModifier> worldsModifiers;
+    private BaseCollection<RealmModel> realmsList => DataStore.i.realm.realmsInfo;
+    private BaseVariable<AboutResponse> realmAboutConfiguration => DataStore.i.realm.playerRealmAbout;
+    private AboutResponse currentConfiguration;
+    
+    public WorldsPlugin()
+    {
+        worldsModifiers = new List<IWorldsModifier>() { new WorldsBlockerModifier() };
+        realmAboutConfiguration.OnChange += RealmChanged;
+        realmsList.OnSet += RealmListSet;
+    }
+
+    private void RealmListSet(IEnumerable<RealmModel> obj)
```

Comment:
if you don't use argument of the function, then you can name it `_`, emphasizing that it is not used inside  
`private void RealmListSet(IEnumerable<RealmModel> _)`